Sources/Rendering/Misc/SynchronizableRenderWindow/ObjectManager/index.d.ts

```diff
@@ -0,0 +1,80 @@
+import { vtkObject } from "../../../../interfaces" ;
+import { Nullable } from "../../../../types" ;
+import { ISynchronizerContext, IViewState } from "..";
+
+export type BuilderFunction = <T extends vtkObject>(type: string, initialProps?: Record<string, unknown>) => Nullable<T>;
+export type UpdaterFunction = (instance: vtkObject, state: IViewState, context: ISynchronizerContext) => void;
+
+export interface IObjectManager {
+  /**
+   * Build a vtkObject.
+   */
+  build: BuilderFunction;
+
+  /**
+   * @param {string} type The type to update 
+   * @param {vtkObject} instance The specific instance 
+   * @param {IViewState} props 
+   * @param {ISynchronizerContext} context 
+   */
+  update(type: string, instance: vtkObject, props: IViewState, context: ISynchronizerContext): Promise<Error | void>;
+
+  /**
+   * Defines a new type handler with the specified builder and updater functions
+   * 
+   * @param {String} type The type you wish to register
+   * @param {BuilderFunction} buildFn The builder function to associate with the type 
```

Comment:
`@param {BuilderFunction} [buildFn] The builder function to associate with the type.`